### ISSUE TITLE
Admin control

### ISSUE THREAD

**Issue by M2168J** [PCGeekBrain/chaverim-update#13]
Can u make it that only a admin can change an account's username/email?

**Comment by PCGeekBrain** [comment]
I can but are you 100% sure that you do not want users to be able to change their email address?

If you want features such as sign in with Google, Facebook or Github we will have to use the emails from these services to log the user in.

**Comment by M2168J** [comment]
Yes. My theory is that if someone wants to play around, he could change his
name  and spam everyone up. Which leads me to the next question - when u
make the excel (spreadsheet) of the calls history for admins/moderators,
besides showing who responded, edited etc.. can you make an additional tab
next to each one showing which "emails" responded?
